packages/opencensus-core/src/trace/instrumentation/types.ts

```diff
@@ -23,13 +23,15 @@ export interface Plugin {
    * @param moduleExports nodejs module exports from the module to patch
    * @param tracer a tracer instance
    * @param version version of the current instaled module to patch
+   * @param basedir module absolute path
    */
-  // tslint:disable:no-any
-  applyPatch(
+  applyPluginPatch(
```

Comment:
Suggestion: `enable` and `disable` to distinguish this from `applyPatch` and `applyUnpatch`.